packages/teams-js/webpack.config.js

```diff
@@ -27,8 +27,8 @@ module.exports = {
       type: 'umd',
       umdNamedDefine: true,
     },
-    //Prevents 'self' object conflict between nodejs and nextjs
-    //globalObject: 'this',
+    //Typically resolves to 'self' unless running in a server side rendered environment
+    globalObject: "typeof self !== 'undefined' ? self : this",
```

Comment:
>globalObject: "typeof self !== 'undefined' ? self : this", [](http://example.com/codeflow?start=4&length=58)

Do we want to wait until we have a C# app checked into the repo that would have broken with the original change before checking in this change to ensure it doesn't break it again?

Reply:
So I do have a C# app that does break with the old changes that I have been using to test against locally. It continues to fail on the old version, and work with this new change, so I'm fairly confident this change should be fine to check in. Unfortunately adding the app to the repo and incorporating tests against it is a lot more involved than the SSR test app, so I won't have that done for a good amount of time. I was hoping to get this change out in the 2.8.0 release on Wednesday. I am also waiting on teams toolkit to see if they are going to add any automation on their end to test for breakages like the one in the past. I should hear back from them by tuesday around noon. Just to note: I didn't plan on checking this pr in until I hear back from them at the earliest.

Reply:
That's great! As long as you have some way to verify it doesn't break the C# app (even if it's not 100% automated yet) that's sufficient.